sdk/storage/Azure.Storage.Common/src/Shared/HashAlgorithmHasher.cs

```diff
@@ -15,7 +14,7 @@ internal class HashAlgorithmHasher : IHasher
     {
         private readonly HashAlgorithm _hashAlgorithm;
 
-        public int HashSize => _hashAlgorithm.HashSize;
+        public int HashSizeInBytes => _hashAlgorithm.HashSize >> 3;
```

Comment:
Curious why we shift by 3? Should this be in our constants?

Reply:
`x >> 3` == `x / 8`. We are just converting from bits to bytes. `HashAlgorithm.HashSize` is in bits for some reason. We convert this to bytes to be consistent with modern .NET tooling. This would probably be clearer if the .NET api used `HashSizeInBits` instead. I can leave a comment to clarify but asking for a constant is asking for `const int BitsInByte = 8` except now we have an even more unwieldy name like `BitToByteBitshiftValue` or something awful when if we just communicate we are converting units it will become very clear.